tarodb/sqlc/migrations/000005_transfers.up.sql

```diff
@@ -42,3 +42,12 @@ CREATE TABLE IF NOT EXISTS asset_deltas (
     
     proof_id INTEGER NOT NULL REFERENCES transfer_proofs(proof_id) 
 );
+
+CREATE TABLE IF NOT EXISTS receiver_proof_transfer_attempts (
+    id INTEGER PRIMARY KEY,
```

Comment:
I think we can remove the id field here (sqlite will add rowid under the hood but we will never care about it).